DaemonRenderer/Source/Include/Vulkan/Utilities/VulkanConfig.hpp

```diff
@@ -28,8 +28,6 @@
 
 #if defined(DAEMON_OS_WINDOWS)
     #define VK_USE_PLATFORM_WIN32_KHR
-#else
```

Comment:
Why has this error been removed? Is the Vulkan code is now cross-platform? If not, the case should be handled in some way or another.